trunk/source/TraceAbstraction/src/de/uni_freiburg/informatik/ultimate/plugins/generator/traceabstraction/tracehandling/StrategyFactory.java

```diff
@@ -129,31 +132,51 @@ public PathProgramCache<L> getPathProgramCache() {
 
 	/**
 	 * Constructs a {@link IRefinementStrategy} that can be used in conjunction with a {@link IRefinementEngine}.
+	 *
+	 * @param counterexample
+	 *            A trace that will be checked for feasibility and for which, if it is infeasible, a refinement result
+	 *            will be constructed.
+	 * @param controlConfigurationSequence
+	 *            Optionally, the sequence of control configurations visited by the trace in the program that is being
```

Comment:
I don't like configurations here (or in the other cases). A configuration of, e.g., a nested word automaton is IIRC the location and the state of the stack. We are only interested in the location (because we can reconstruct, e.g. the stack from the counterexample / word).

Reply:
See Matthias' comment and the discussion about the naming above.